### DISCUSSION TITLE
2.3.10 fleet

### DISCUSSION THREAD

**Discussion by tommorgan365** [Security-Onion-Solutions/securityonion#2053] [Unsupported Versions] [answered]
getting 404 error after sign into fleet post 2.3.10 update.  so-status shows everything ok on fleet.  Suggestions for troubleshooting?

**Comment by defensivedepth** [comment]
Restart Fleet by running `so-fleet-restart --force`  -- are there any errors?

You can also check the Fleet logs at `/opt/so/log/fleet/status.log`

**Reply by tommorgan365** [reply]
Hello, distributed environment with separate fleet server.  run the so-fleet-restart --force on fleet server or manager ?

**Reply by tommorgan365** [reply]
I see a lot of these error messages:
*1499 open() "/opt/socore/html/kolide.launcher.QueryTarget/GetTargets" failed (2: No such file or directory), client

**Reply by tommorgan365** [reply]
I did notice that my old url for fleet pointed to the fleet server , now when I go to it refers back to manager.

**Reply by defensivedepth** [reply]
Will look into this further.

**Reply by defensivedepth** [reply]
@tommorgan365 Did you initially install Fleet on the Manager, and then setup a Fleet Standalone Node?

Also, can you post the sanitized output of:  `sudo salt-call pillar.get global`  (run on the Manager)

**Reply by tommorgan365** [reply]
Yes, did install on manager then setup fleet standalone

local:
    ----------
    airgap:
        False
    cortexkey:
        xx
    cortexorgname:
        SecurityOnion
    cortexorguserkey:
        xx
    cortexplaysecret:
        xx
    fleet_custom_hostname:
        None
    fleet_hostname:
        fleethostname
    fleet_ip:
        fleetip
    fleet_manager:
        True
    fleet_node:
        True
    fleet_packages-timestamp:
        2020-10-22-15:48
    fleet_packages-version:
        3
    hivekey:
        ***	
   hiveplaysecret:
        ***
    hnmanager:
        managernetwork
    ids:
        Suricata
    imagerepo:
        securityonion
    managerip:
        managerip
    managerupdate:
        1
    mdengine:
        ZEEK
    ntpserver:
    pipeline:
        redis
    proxy:
    sensoronikey:
        ***
    soversion:
        2.3.10
    url_base:
        ipaddress of manager
    wazuh:

**Comment by defensivedepth** [comment, marked as answer]
I believe the following will fix the issue, please let me know:

Edit line 101 in `/opt/so/saltstack/default/salt/nginx/etc/nginx.conf` on the Manager - replace

	server_name  {{ url_base }};

with

	server_name  {{ main_ip }};

Then restart nginx on both the Manager, then on the Fleet Standalone node (`sudo so-nginx-restart --force`)

**Reply by tommorgan365** [reply]
Hello, that worked - thanks.

**Reply by defensivedepth** [reply]
Great to hear! This will be permanently fixed in our next release.
